spoon-client/src/main/java/com/squareup/spoon/Spoon.java

```diff
@@ -8,11 +8,16 @@
 import android.os.Looper;
 import android.util.Log;
 import android.view.View;
+
```

Comment:
kill this line (installing https://github.com/square/java-code-styles should do this for you)